doc/src/Howtorun.tex

```diff
@@ -4,31 +4,32 @@ \section{Installation}
 This section of the manual describes how to install the TEAL plug-in and use it from within RAVEN \cite{RAVEN}.
 
 \subsection{Installing the plug-in}
-The TEAL plugin is distributed with RAVEN. If you just want to use the plugin and are not interested in developing additional features or bug-fixes for it, just make sure that you have access to the plugin repository and you will automatically get it when installing RAVEN.
-For more information, please refere to the RAVEN manual.
+The TEAL plug-in is distributed with RAVEN. If you just want to use the plug-in, but are not interested in developing additional features or bug-fixes for it, make sure that you have access to the plug-in repository and you will automatically get it when installing RAVEN.
+For more information, please refer to the RAVEN manual.
 
 \subsection{Accessing the plug-in from within RAVEN}
-The plugin can be accessed as a special subtype of the External model. The syntax is given in Listing \ref{lst:TEALfromRAVEN}.
+The plug-in can be accessed as a special subtype of the external model. The syntax is given in Listing \ref{lst:TEALfromRAVEN}, where "CashFlowinput.xml" is the text file containing the TEAL code.
```

Comment:
I think `ExternalModel` is more accurate than `external model`, since we're describing the RAVEN entity. Possibly `\texttt{ExternalModel}` would be most clear.